toolchain/lowering/lowering_context.h

```diff
@@ -77,13 +77,13 @@ class LoweringContext {
   llvm::LLVMContext* llvm_context_;
   std::unique_ptr<llvm::Module> llvm_module_;
 
-  // The input Semantics IR.
-  const SemanticsIR* const semantics_ir_;
+  // The input SemIR:: IR.
```

Comment:
I went with just "SemIR" here instead, rather than "semantics IR"
https://github.com/carbon-language/carbon-lang/pull/3138/commits/93cf4d8e2e7a58c54fa4f2f567c54e73ba64662e